utils/get_data.py

```diff
@@ -17,11 +21,37 @@ def get_data(asset_type, payload):
         BASE_URL, asset_type,
         payload_encoded)
 
-    # # Get the JSON
+    # Get the JSON
     response = urllib2.urlopen(url)
     r = json.load(response)
     results = r["results"]
-    # count = r["count"]
 
     # Return the results
     return results
+
+
+def get_bbox(iface):
```

Comment:
What is the reason that this bbox function is in this module, while the other bbox related functions are in the geometry.py module?